### PR TITLE
Allow Steel Jerrycans to be used as vehicle tanks.

### PR DESCRIPTION
**Pull request by DaliTimelord**
I was particularly frustrated by how a plastic jerrycan can be used, but a steel jerrycan cannot be used.

### CODE REVIEW

File: data/json/items/containers.json [hunk 1]
```diff
@@ -483,7 +483,7 @@
         "bashing" : 6,
         "to_hit" : -4,
         "contains" : 240,
-        "material" : ["steel"],
+        "material" : "steel",
```
Comment (mugling) [inline data/json/items/containers.json]: spurious change

**Comment by mugling** [comment]
It may be best to complete work on generic vehicle tanks before adding further tanks

File: data/json/items/containers.json (hunk 1 above)
Comment (DaliTimelord) [inline data/json/items/containers.json]: Was noticed when referencing.
Items with a single material type did not use brackets, but those with multiple did.

Comment (mugling) [inline data/json/items/containers.json]: Try to keep line counts as low as possible for each PR by avoiding any unrelated or unnecessary changes

Comment (DaliTimelord) [inline data/json/items/containers.json]: Thank you for advice! Will keep that in mind in the future.

Comment (mugling) [inline data/json/items/containers.json]: In new JSON code you should always prefer the array form
